static/css/onboarding.scss

```diff
@@ -259,3 +261,36 @@ body {
 .active-slide-4 #done {
   display: inline-block;
 }
+
+/* for smaller screen sizes */
+@media screen and (max-width: 768px) {
+  .slide {
+    height: $slide-height-small;
+    width: $slide-width-small;
+
+    .slide-image {
+      background-size: contain;
+      background-repeat: no-repeat;
+      background-position: center;
+      flex: 0 0 200px;
+    }
+
+    .slide-content {
+      flex: 0 0 160px;
+    }
+  }
+
+  #skip {
+    margin-left: - ($slide-width-small / 2) + 20;
+    margin-top: ($slide-height-small / 2) - 3;
+  }
+
+  #prev {
+    margin-left: - ($slide-width-small / 2) - ($next-prev-size / 2);
+}
+
+  #next,
+  #done, {
```

Comment:
@niharikak101 i didn't notice this at the time, but you have an extra comma on this line! I fixed it in the subsequent PR